### PR TITLE
docs: favicon

### PR DESCRIPTION
**Pull request by mainframev**
Storybook: https://orbit-feat-favicon.surge.sh

### CODE REVIEW

File: docs/src/images/copy.svg [hunk 1]
```diff
@@ -1,4 +1 @@
...
-<path d="M5 5C5 4.44772 4.55228 4 4 4H2C0.895431 4 0 4.89543 0 6V18C0 19.1046 0.89543 20 2 20H12C13.1046 20 14 19.1046 14 18C14 17.4477 13.5523 17 13 17H12.75C12.3358 17 12 17.3358 12 17.75C12 17.8881 11.8601 18 11.6875 18H2.3125C2.13991 18 2 17.8881 2 17.75V6.25C2 6.11193 2.13991 6 2.3125 6H4C4.55228 6 5 5.55228 5 5Z" fill="#252A31"/>
-</svg>
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M15.586 0a1 1 0 0 1 .707.293l3.414 3.414a1 1 0 0 1 .293.707V14a2 2 0 0 1-2 2H8a2 2 0 0 1-2-2V2a2 2 0 0 1 2-2h7.586zm2.101 14c.173 0 .313-.112.313-.25V5.16c0-.069-.07-.16-.156-.16H16c-.69 0-1-.448-1-1V2.16c0-.069-.074-.16-.16-.16H8.313C8.14 2 8 2.112 8 2.25v11.5c0 .138.14.25.313.25h9.374z" fill="#252A31"/><path d="M5 5a1 1 0 0 0-1-1H2a2 2 0 0 0-2 2v12a2 2 0 0 0 2 2h10a2 2 0 0 0 2-2 1 1 0 0 0-1-1h-.25a.75.75 0 0 0-.75.75c0 .138-.14.25-.313.25H2.313C2.14 18 2 17.888 2 17.75V6.25c0-.138.14-.25.313-.25H4a1 1 0 0 0 1-1z" fill="#252A31"/></svg>
```
Comment (silvenon) [inline docs/src/images/copy.svg:1]: What happened here?

**Review by silvenon — COMMENTED** [review]
`<head>` includes extra `apple-touch-icon` icons, is there any way to add only `180×180`? Also, the manifest contains extra sizes.

If it's not possible to configure, after reverting the SVG changes below you can merge.

**Comment by mainframev** [comment]
> `<head>` includes extra `apple-touch-icon` icons, is there any way to add only `180×180`? Also, the manifest contains extra sizes.
> 
> If it's not possible to configure, after reverting the SVG changes below you can merge.

it's possible

**Review by silvenon — COMMENTED** [review]
After fixing the Prettier problem consider the PR approved if there's no way to generate the `<link rel="apple-touch-icon">` tag as in the blog post.

**Pull request merged**